Makefile

```diff
@@ -186,12 +186,29 @@ run-test-multikueue-e2e-%: FORCE
 	@echo Running multikueue e2e for k8s ${K8S_VERSION}
 	E2E_KIND_VERSION="kindest/node:v$(K8S_VERSION)" KIND_CLUSTER_NAME=$(KIND_CLUSTER_NAME) CREATE_KIND_CLUSTER=$(CREATE_KIND_CLUSTER) ARTIFACTS="$(ARTIFACTS)/$@" IMAGE_TAG=$(IMAGE_TAG) ./hack/multikueue-e2e-test.sh
 
+PROJECT_BIN := $(shell dirname $(abspath $(lastword $(MAKEFILE_LIST))))/bin
```

Comment:
We have already defined in https://github.com/kubernetes-sigs/kueue/blob/1a482fdff85aade8f5f278919a7cb5a81b98615e/Makefile#L314.

Reply:
I tried using PROJECT_BIN but the target was returning error for undefined variable. I'll try again.